### DISCUSSION TITLE
Edit code of sections

### DISCUSSION THREAD

**Discussion by domoritz** [home-assistant/frontend#22700] [Dashboard] [answered]
I would like to be able to edit the code of a section. The reason is that I have existing dashboards with custom stacked components that I now want to convert to sections. Being able to edit the code will make it possible for me to convert the existing components. I would also be able to copy a section from one dashboard to another.

**Comment by karwosts** [comment]
You can edit the yaml for the entire view if you click on the View Configuration dialog, there's an overflow menu in the top right for yaml mode.

**Reply by domoritz** [reply, marked as answer]
Oh, that actually exists for sections as well. I didn't see it earlier. Thanks!

<img width="614" alt="Screenshot 2024-11-07 at 19 50 12" src="https://github.com/user-attachments/assets/9fe92cf4-1cc3-4196-861d-a5acec8deea6">
